cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DistributedStorageProviderNamespaceAdmin.java

```diff
@@ -106,8 +106,9 @@ public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreExcep
 
     try (HBaseAdmin admin = new HBaseAdmin(hConf)) {
       if (!tableUtil.hasNamespace(admin, hbaseNamespace)) {
-        throw new IOException(String.format("Custom mapped HBase namespace doesn't exist %s for namespace %s",
-                                            hbaseNamespace, namespaceMeta.getName()));
+        throw new IOException(String.format("HBase namespace '%s' specified for custom mapped namespace '%s' does not" +
```

Comment:
Sure, I can remove it, however documentation for custom mapped namespaces does exist http://docs.cask.co/cdap/4.1.1/en/developers-manual/building-blocks/namespaces.html#custom-mapping-of-storage-providers